src/abstractdataframe/abstractdataframe.jl

```diff
@@ -530,19 +530,24 @@ julia> describe(df, :min, :sum => sum)
 │ 2   │ x        │ 0.1 │ 5.5 │
 │ 3   │ y        │ 'a' │     │
 
-julia> describe(df, :min, :sum => sum, cols=:x)
+julia> describe(df, cols=:x, :min, sum => :sum)
```

Comment:
I'd rather keep `cols` at the end, that's more common.
```suggestion
julia> describe(df, :min, sum => :sum, cols=:x)
```

Reply:
I made it on purpose as @matthieugomez wanted the `cols` argument to go second (and kwarg can be placed anywhere). So - which style do we want to settle to?

Reply:
(I am accepting the suggestion, but we might revert it if we want)

Reply:
I mean it's fine if people want to put it first (that's probably a good idea), but for clarity in the reference docs I'd put it last, like we always do for keyword arguments.